packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -4,28 +4,51 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { Aliases, AuthInfo } from '@salesforce/core';
-import { ForceConfigGet } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
-
+import {
+  Aliases,
+  AuthInfo,
+  ConfigAggregator,
+  ConfigFile
+} from '@salesforce/core';
+import * as path from 'path';
+import { getRootWorkspacePath, hasRootWorkspace } from './index';
 export class OrgAuthInfo {
   public static async getDefaultUsernameOrAlias(
     vscodePath: string
   ): Promise<string | undefined> {
-    const forceConfig = await new ForceConfigGet().getConfig(
-      vscodePath,
-      'defaultusername'
-    );
-    return forceConfig.get('defaultusername');
+    if (hasRootWorkspace()) {
+      const rootPath = path.join(getRootWorkspacePath(), '/');
```

Comment:
Sorry I wasn't clear. Once you use `path.join` you shouldn't need to add a `/` to the path, that utility should build the correct paths for you. So in line 20 you would only need to call `getRootWorkspacePath()` and then you just use `path.join` for creating the `rootFolder` value.

Reply:
No worries, I'd actually kept it like that originally with only `getRootWorkspacePath()` but I was seeing some weird behavior. When it hit the `path.join` for building the `rootFolder` path, it was getting built without the `/`. So when it was looking for the `rootFolder` path, the result was `/Users/a.jha/Documents/repos/dreamhouse-sfdx.sfdx/sfdx-config.json`. Should I be building the path in a different way?

Reply:
That's weird, I changed it to 
```
const rootPath = getRootWorkspacePath();
const rootFolder = path.join(rootPath, '.sfdx');
```
and `rootFolder` is set to `"/Users/lcamposguajardo/Desktop/testNewOrgPicker/.sfdx"`. Do you have any other changes in your local that could be causing the result to be different ?

Reply:
Ah I think it must not have compiled properly because it still thought the , was a +. All good now, thanks.